Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -3912,6 +4136,256 @@ def screenshot_url_command(client: Client, args: dict[str, Any]) -> CommandResul
     )
 
 
+@metadata_collector.command(
+    command_name="silentpush-add-feed",
+    inputs_list=ADD_FEED_INPUTS,
+    outputs_prefix="SilentPush.AddFeed",
+    outputs_list=ADD_FEED_OUTPUTS,
+    description="This command add the new feed",
+)
+def add_feed_command(client: Client, args: dict[str, Any]) -> CommandResults | dict:
+    """
+    Command handler for adding new feeds.
+
+    Args:
+        client (Client): SilentPush API client instance.
+        args (Dict[str, Any]): Command arguments, must include 'name' and 'type' key.
+
+    Returns:
+        CommandResults: JSON response of added feed.
+    """
+
+    name = args.get("name")
+    if not name:
+        raise ValueError("Name is required")
+
+    feed_type = args.get("type")
+    if not feed_type:
+        raise ValueError("Type is required")
+
+    result = client.add_feed(args)
+
+    return CommandResults(
+        outputs_prefix="SilentPush.AddFeed",
+        outputs_key_field="name",
+        outputs=result,
+        readable_output=tableToMarkdown("SilentPush Add Feed", result),
+        raw_response=result,
+    )
+
+
+ADD_FEED_TAGS_INPUTS = [
+    InputArgument(
+        name="feed_uuid",
+        description="Never return query metadata, even if original request did include metadata.",
+    ),
+    InputArgument(
+        name="tags",
+        description="Comma separated tags to be updated to the feed.",
+    ),
+]
+
+ADD_FEED_TAGS_OUTPUTS = [
+    OutputArgument(name="created_or_updated", description="List of tags that have been created or updated to the feed.")
+]
+
+
+@metadata_collector.command(
+    command_name="silentpush-add-feed-tags",
+    inputs_list=ADD_FEED_TAGS_INPUTS,
+    outputs_prefix="SilentPush.AddFeedTags",
+    outputs_list=ADD_INDICATORS_OUTPUTS,
+    description="This command add indicators to the feed",
+)
+def add_feed_tags_command(client: Client, args: dict[str, Any]) -> CommandResults | dict:
+    """
+    Command handler for adding new feed tags.
+
+    Args:
+        client (Client): SilentPush API client instance.
+        args (Dict[str, Any]): Command arguments, must include 'feed_uuid' key.
+
+    Returns:
+        CommandResults: JSON response of added tags.
+    """
+    feed_uuid = args.get("feed_uuid")
+    if not feed_uuid:
+        raise ValueError("Feed UUID is required")
+
+    if not args.get("tags"):
+        raise ValueError("Tags name is required")
+
```

Comment:
if the arg is set to required in the yml, no need for this check, the command will not run without it.